For the annotation of the C. remanei genome, proteins from _C. elegans_ ([PRJNA13758](https://www.ebi.ac.uk/ena/browser/view/PRJNA13758)), _C. briggsae_ ([PRJNA10731](https://www.ebi.ac.uk/ena/browser/view/PRJNA10731)), and _C. latens_ ([PRJNA248912](https://www.ebi.ac.uk/ena/browser/view/PRJNA248912)) from [WormBase 260](https://downloads.wormbase.org/releases/WS260/species/) have been used with MAKER2 ([Holt and Yandell et al., 2011](https://doi.org/10.1186/1471-2105-12-491)); RNA-seq from the group's previous C. remanei studies ([SRX3014311](https://www.ebi.ac.uk/ena/browser/view/SRX3014311?show=reads) and [SRP049403](https://www.ebi.ac.uk/ena/browser/view/SRP049403?show=reads)) with [BRAKER2](https://github.com/Gaius-Augustus/BRAKER), and ab initio gene prediction from GeneMark-ES ([Ter-Hovhannisyan et al., 2008](https://doi.org/10.1101/gr.081612.108)). Annotations from BRAKER2, MAKER2, and GeneMark-ES were combined with [EVidenceModeler](https://evidencemodeler.github.io/) ([Haas et al., 2008](https://dx.doi.org/10.1186%2Fgb-2008-9-1-r7)). CDS from EVidenceModeler were used to train AUGUSTUS ([Stanke et al., 2006](https://doi.org/10.1093/nar/gkl200)), and then CDS obtained from single-molecule long-read RNA sequencing (Iso-Seq) data ([SRX6989868](https://www.ebi.ac.uk/ena/browser/view/SRX6989868?show=reads) and [SRX6989871](https://www.ebi.ac.uk/ena/browser/view/SRX6989871?show=reads)) were also added as hints for the final annotation with AUGUSTUS. See details of the annotation and repeat masking pipelines in [Teterina et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32111628/).
