According to [Dayi, M., et al. (2021)](https://doi.org/10.1038/s41598-021-85967-z), three de novo assemblers were used to generate initial assemblies. The Nanopore reads were assembled with [Flye](https://github.com/fenderglass/Flye) or [Canu](https://canu.readthedocs.io/), both followed by base correction by Illumina DNA reads using [Pilon](https://github.com/broadinstitute/pilon). [Spades](https://github.com/ablab/spades) (v.3.7.1) was separately used to generate a hybrid assembly of Nanopore, Illumina pair-end and mate-pair reads. The three assemblies were merged using [MetaAssembler](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0764-4) (v.1.5) with the Flye assembly as a reference. [Haplomerger2](https://github.com/mapleforest/HaploMerger2) (20151106) was run on the merged assembly to remove remaining haplotypic sequences. Further base corrections were performed by ICORN253.
