The nuclear genome assembly was annotated using [Braker v2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7787252/). As no RNA-seq data were available, hints were generated for Braker using the [ProtHint](https://github.com/gatech-genemark/ProtHint) pipeline; spliced alignments were generated by mapping proteins from [OrthoDB](https://www.orthodb.org/) Metazoan protein database, from which evidence was used as an input to Braker.

The mtDNA genome sequence was initially annotated using [MITOS](https://pubmed.ncbi.nlm.nih.gov/22982435). The annotation was further improved manually by comparing sequence alignments and GFF3 annotation files from C. johnstoni with the closely related filarial nematodes [_L. loa_](https://parasite.wormbase.org/Loa_loa_prjna246086/Info/Index/), [_D. immitis_](https://parasite.wormbase.org/Dirofilaria_immitis_prjeb1797), [_A. viteae_](http://parasite.wormbase.org/Acanthocheilonema_viteae_prjeb1697), [_B. malayi_](https://parasite.wormbase.org/Brugia_malayi_prjna10729), [_O. ochengi_](http://test.parasite.wormbase.org/Onchocerca_ochengi_prjeb1204), [_O. volvulus_](https://parasite.wormbase.org/Onchocerca_volvulus_prjeb513), [_W. bancrofti_](https://parasite.wormbase.org/Wuchereria_bancrofti_prjna275548).
