The full assembly process is outline in [Casasa et al., (2021)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7826178/). Paired-end reads between 150- and 500-bp in length were trimmed using [Cutadapt](https://journal.embnet.org/index.php/embnetjournal/article/view/200/479) and error corrected with [Reckoner](https://academic.oup.com/bioinformatics/article/33/7/1086/2843893). To check insert size, a preliminary single-end assembly was created with [Minia](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4184257/). [SPAdes](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3342519/) was then used to fully assemble the nuclear genome, and scaffolds that were identified as contamination, at low coverage (<1×), and < 500 bp in length were removed. [PlasmidSPAdes](https://pubmed.ncbi.nlm.nih.gov/27466620/) was used to assemble the mitochondrial genome.
