For this assembly, a _Heterodera schachtii_ IRS (a Dutch beet cyst nematode population) was used.

According to [Joris J. M. van Steenbrugge et. al. (2022)](https://onlinelibrary.wiley.com/doi/10.1111/mec.16505), raw long [Oxford Nanopore](https://nanoporetech.com/) reads were corrected to, in essence, merge haplotypes using the correction mode in [CANU](https://doi.org/10.1101/gr.215087.116). Using [wtdgb2](https://doi.org/10.1038/s41592-019-0669-3) version 2.3, multiple initial genome assemblies were generated. The contigs from the assemblies were then improved using [finishersc](https://doi.org/10.1093/bioinformatics/btv280) version 2.1. Gaps in the assemblies were then filled using [gapfiller](https://doi.org/10.5281/zenodo.4627096) version 1.0. The resulting assembly was polished with ONT reads using [MEDAKA](https://github.com/nanoporetech/medaka) v1.4.1, followed by five iterations of polishing with [pilon](https://doi.org/10.1371/journal.pone.0112963) version 1.23 using [Illumina](https://emea.illumina.com/systems/sequencing-platforms/novaseq.html) short reads.
