This genome was submitted to ENA on July 18, 2023 by [LBMC UMR5239, ENS de Lyon](https://www.ens-lyon.fr/LBMC) and has been imported from there. 10 strains of the auto-pseudogamous species _M. belari_ populations have been pooled and sequenced using both short- (Illumina HiSeq 4000) and long-read sequencing technology (Oxford Nanopore Techonologies - MinION). The reads were polished and assembled using [Canu](https://canu.readthedocs.io/en/latest/quick-start.html), [Flye](https://github.com/fenderglass/Flye), [Quickmerge](https://github.com/mahulchak/quickmerge), [Pilon](https://github.com/broadinstitute/pilon), [Medaka](https://github.com/nanoporetech/medaka) and [purge_haplotig_onlyforannotation](https://bitbucket.org/mroachawri/purge_haplotigs/src/master/).

The new improved genome has replaced the previous _M. belari_ genome [GCA_900631915.1](https://ncbi.nlm.nih.gov/datasets/genome/GCA_900631915.1/) ([Grosmaire M, et al., 2019] from [LBMC UMR5239, ENS de Lyon](https://www.ens-lyon.fr/LBMC)(https://europepmc.org/article/MED/30872523)) in WormBase ParaSite Release 19.
