The gene predictions were made by the <a href="http://www.sanger.ac.uk/research/projects/parasitegenomics/">Parasite Genomics group at the Wellcome Trust Sanger Institute</a> and WormBase, as part of the 50 Helminth Genomes project. An in-house pipeline was developed that used <a href="http://europepmc.org/abstract/MED/22192575">MAKER</a> to generate high-quality annotations by integrating evidence from multiple sources: ab initio gene predictions from <a href="http://europepmc.org/abstract/MED/16845043">AUGUSTUS</a>, <a href="http://europepmc.org/abstract/MED/18757608">GeneMark-ES</a>, and <a href="http://europepmc.org/abstract/MED/15144565">SNAP</a>; projected annotation from <em>C.&nbsp;elegans</em> (using <a href="http://europepmc.org/abstract/MED/21653517">GenBlastG</a>) and the taxonomically nearest reference helminth genome (using <a href="http://europepmc.org/abstract/MED/21306991">RATT</a>); and ESTs, mRNAs and proteins from related organisms aligned to the genome using BLAST, with refinement of alignments using <a href="http://europepmc.org/abstract/MED/15713233">Exonerate</a>.
