RNA-sequencing (RNA-seq) data were aligned to the assembly and used to infer transcripts. The RNA-seq mappings were also used in [braker](https://github.com/Gaius-Augustus/BRAKER) to train species parameters and generate an initial set of annotations. Proteomes of _Bursaphelenchus xylophilus_ and _Caenorhabditis elegans_ were used as homology guides to pick the best transcripts for each putative locus using [mikado](https://mikado.readthedocs.io/en/stable/) and were also used to train [MAKER2](https://www.yandell-lab.org/software/maker.html). Finally, [MAKER2](https://www.yandell-lab.org/software/maker.html) was invoked to generate a final set of gene annotations. The training models for the prefiction were also based on the manual curation of 975 genes. Full details can be found on [Lai, C.-K., et al., 2023](https://onlinelibrary.wiley.com/action/showCitFormats?doi=10.1111%2F1755-0998.13752)
