Evidence-based gene annotations were generated based on protein homology and transcriptome data using the PPCAC software (version 1.0; [Athanasouli et al. 2020](https://europepmc.org/article/MED/33045985); [Rödelsperger 2021](https://europepmc.org/article/MED/33765927)). Specifically, the community-curated gene annotations of P. pacificus (El Paco gene annotations, version 3; [Athanasouli et al. 2020](https://europepmc.org/article/MED/33045985)) and a mixed-stage transcriptome assembly of P. giblindavisi ([Rödelsperger et al. 2018](https://doi.org/10.1016/j.cub.2018.07.041)) were transferred to the new P. giblindavisi genome assembly by alignment with the software [exonerate](https://github.com/nathanweeks/exonerate) (version 2.2.0). For each 100-bp window, one representative gene model was chosen by selecting the gene model with the longest open reading frame (minimum protein length 60 amino acids, at least 3 exons). All other alternative gene models within a 100-bp window were discarded.
