The gene predictions were made by the [Department of Biology at the Indiana University]([http://www.sanger.ac.uk/research/projects/parasitegenomics/](https://biology.indiana.edu/). An in-house pipeline was developed that used [MAKER](http://europepmc.org/abstract/MED/22192575) to generate high-quality annotations by integrating evidence from multiple sources: protein sequences from [UniProt](https://www.uniprot.org/), the hidden Markov model (HMM) computed from [GeneMark](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC147337/) and the [BUSCO](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850278/) generated [AUGUSTUS](https://pubmed.ncbi.nlm.nih.gov/14534192) profile. Predicted genes from the MAKER output were used to train and run AUGUSTUS to produce the final annotation. 
