The draft genome assembly was produced by [The H. schachtii genome sequencing consortium](https://www.plantsci.cam.ac.uk/research/sebastianevesvandenakker). As described by [Shahid Siddique et al., (2022)](https://europepmc.org/article/MED/36261416), the assembly uses a combination of Pacific Biosciences sequencing, and Illumina sequencing. PacBio reads were assembled using [wtdbg2](https://github.com/ruanjue/wtdbg2). The assembly was submitted to decontamination with [BlobTools](https://doi.org/10.12688/f1000research.12232.1), upgrade with [FinisherSC](https://doi.org/10.1093/bioinformatics/btv280), scaffolding with [SSPACE_longread](https://github.com/Runsheng/sspace_longread) and [gapFinisher](https://github.com/kammoji/gapFinisher), polishing with PacBio's smrttools and [Pilon](https://doi.org/10.1371/journal.pone.0112963) using the raw PacBio reads and the Illumina reads.
