Four runs of initial prediction of protein-coding genes were done with [BRAKER2](https://github.com/Gaius-Augustus/BRAKER) in parallel, using four different modes:
- ab initio, without softmasking
- with input data from RNA-seq only, without softmasking
- with input data from Clade IV proteomes, without softmasking
- with input data from both RNA-seq and Clade IV proteomes, with softmasking.
  
The RNA-seq data came from two developmental stages, infectious juveniles (IJs) and mixed-sex adults; these stages were chosen to try getting data from as wide a range of genes as reasonably possible from two biological replicates.  The Clade IV protein data came from previously published proteomes of these related nematodes, taken from the WBPS17 release of ParaSite: _Bursaphelenchus okinawaensis_, _Panagrellus redivivus_, _Rhabditophanes sp. KR3021_, _Steinernema carpocapsae_, _Steinernema feltiae_ (two alternative proteomes), _Steinernema glaseri_, _Steinernema monticolum_, _Steinernema scapterisci_, and _Strongyloides ratti_. The results of multiple [BRAKER2](https://github.com/Gaius-Augustus/BRAKER) runs were merged with [TSEBRA](https://github.com/Gaius-Augustus/TSEBRA) while enforcing retention of all transcripts. Structurally redundant transcripts (transcripts belonging to a single merged gene, and encoding either exactly the same protein or subsets of exactly the same protein) were then reduced to a single transcript encoding the largest possible protein. The annotation was generated and provided by [Erich Schwarz](https://cals.cornell.edu/erich-schwarz) and collaborators at [Cornell University](https://cals.cornell.edu/). 
