The genome assembly was produced by the [Mitreva laboratory of the Division of Infectious Diseases at theWashington University School of Medicine in St. Louis](https://infectiousdiseases.wustl.edu/people/makedonka-mitreva/). The assembly uses Illumina HiSeq platform (2X100bp) paired-end sequencing complemented by PacBio RS II P5-C3/P6-C4, 20-kb long-read sequencing. [ALLPATHS-LG](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3029755) was used to assemble the Illumina reads. [SSPACE-standard v3.0](https://pubmed.ncbi.nlm.nih.gov/21149342) and [SSPACE-longread v1.1](https://pubmed.ncbi.nlm.nih.gov/24950923) were sequentially run using the Illumina and PacBio reads, respectively, to scaffold the assembled contigs. [Gapfiller v1.10](https://pubmed.ncbi.nlm.nih.gov/22731987) and [PBJelly v15.8.24](https://pubmed.ncbi.nlm.nih.gov/23185243) were used to close gaps, and the resulting assembly was error-corrected using [Pilon v1.20](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4237348/) and screened for contaminants using [blobtools v0.9.19](https://blobtools.readme.io/). Mitochondrial genomes (mtDNA) were assembled with [NOVOPlasty v2.6.3](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5389512/) using the Illumina fragment reads.
