The gene predictions were made by the [Mitreva laboratory of the Division of Infectious Diseases at theWashington University School of Medicine in St. Louis](https://infectiousdiseases.wustl.edu/people/makedonka-mitreva/). The nuclear genome was annotated using the [MAKER pipeline v2.31.8](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7528651/#msz204-B43). Publicly available and in-house RNA-Seq studies, mRNA and EST sequences and protein sequences from closely related species were used by [BRAKER](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6078167) and MAKER pipelines as supporting evidence. The best-supported gene models were chosen based on [Annotation Edit Distance](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2653490/). [PANNZER2](https://pubmed.ncbi.nlm.nih.gov/25653249) and [sma3s v2](https://pubmed.ncbi.nlm.nih.gov/25653249) were employed to name gene products. Mitochondrial genomes were annotated using [MITOS2](https://pubmed.ncbi.nlm.nih.gov/22982435).
