The full assembly procedure is outlined in [Casasa et al., (2021)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7826178/). Raw reads were adapter-trimmed and quality-filtered using Trimmomatic. Cleaned reads were assembled using SPAdes and scaffolds that were identified as contamination, at low coverage (<1×), and < 500 bp in length were removed. From the assembled contigs, 10-kb fragments (sliding over 5 kb) were extracted and 4-mer frequencies for each fragment were computed. This oligonucleotide frequency data was mapped to 3D space using Rtsne and clustered using DBSCAN. Each scaffold was assigned to the cluster with the highest number of fragments derived from that scaffold. Scaffold sequences from each cluster were aligned to NCBI nt database using BLAST to obtain the corresponding taxonomic associations. This distinguished nematode sequences from those of bacterial contaminants, which were subsequently removed from the assembly. Using a Burrows–Wheeler aligner, the cleaned reads were mapped back to the assembly scaffolds.
