The full annotation process is described in [Lee et al., (2023)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10450198/). 
Single-worm transcriptome reads were mapped to the corresponding genome assemblies using [STAR](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4631051/) (ver. 2.7.7a). The gene models were predicted using [BRAKER2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7787252/) (ver. 2.1.6; option: â€“etpmode) with proteomes and RNA-seq mappings as evidence hints. Transcript predictions were mapped to the reference genome using [Minimap2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6137996/) (-ax splice), converted to gff format and compared against the reference proteome using [Gffcompare](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7222033/) (ver. v0.11.2).
