A genome assembly pipeline was used to generate a high-quality assembly from long sequencing reads. [Canu](https://pubmed.ncbi.nlm.nih.gov/28298431/) was used for error correction, followed by assembly with [wtdbg2](https://github.com/ruanjue/wtdbg2). The assembly was assessed for contamination using [BlobTools](https://f1000research.com/articles/6-1287/v1). A BLASTn approach was followed to further decontaminate the assembly. Contaminant contigs were removed. The assembly was then polished using PacBio reads with Arrow and [Pilon](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0112963). The assembly was also phased using [Freebayes](http://arxiv.org/abs/1207.3907), [Whatshap](https://doi.org/10.1101/085050), and [Bcftools](https://pubmed.ncbi.nlm.nih.gov/33590861). All scripts used to conduct the analysis can be found [here](https://github.com/peterthorpe5/G.pallida_newton_assembly). The data was directly submitted to WormBase ParaSite by [Dr Peter Thorpe](https://www.dundee.ac.uk/people/peter-thorpe). Awaiting publication.
