[BRAKER2](https://academic.oup.com/nargab/article/3/1/lqaa108/6066535) was used for gene model prediction, assisted by short-read RNAseq data from [Cotton et al., (2014)](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-3-r43). The resulting BRAKER predicted GFF file and the GeneMark-ET GFF files were passed to [Funannotate](https://funannotate.readthedocs.io/en/latest/). DIAMOND_BLASTP search against Swiss-prot was also given as evidence. The genome-guided RNAseq assembly was also passed to Funannotate, which runs PASA. The gene models were further refined in the “update” stage of Funannotate using a combination of PASA and [Stringtie](https://doi.org/10.1038/nbt.3122) (Pertea et al., 2015). All scripts used to conduct the analysis can be found [here](https://github.com/peterthorpe5/G.pallida_newton_assembly). The gene models was directly submitted to WormBase ParaSite by [Dr Peter Thorpe](https://www.dundee.ac.uk/people/peter-thorpe). Awaiting publication.
