Gene models were provided by the [Evolutionary Biology Group, Faculty of Biology, Adam Mickiewicz University](https://evobio.home.amu.edu.pl/). Different _ab initio_/evidence-based gene prediction were combined to obtain gene models: [Geneid](https://genome.crg.es/software/geneid/), [glimmerHMM](https://ccb.jhu.edu/software/glimmerhmm/),  [GeneMarkES](http://exon.gatech.edu/), [geneMarkET](http://exon.gatech.edu/) and [SNAP](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-5-59), using a combination of external RNASeq data, training gene models produced by PASA pipeline and PASA-derived transcript evidence. A very thorough procedure had been followed to combine the output of the above methods and produce the final gene models. More details can be found [here](https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1111%2Fmec.15421&file=mec15421-sup-0001-AppendixS1.pdf).
